src/test/java/net/snowflake/client/jdbc/DatabaseMetaDataLatestIT.java

```diff
@@ -712,4 +712,30 @@ public void testHandlingSpecialChars() throws SQLException {
     assertTrue(resultSet.next());
     assertEquals("COLA", resultSet.getString("COLUMN_NAME"));
   }
+
+  /**
+   * This tests that the driver doesn't add quotes to the database name if the application has
+   * already added them. This fixes a bug where the show schemas command was returning empty results
+   * since the database name was being double-quoted.
+   *
+   * @throws SQLException
+   */
+  @Test
```

Comment:
This test is failing on the GithubActions automated test run because of a permissions issue with creating new databases on the account. Please add this tag to the test to get around this:
```suggestion
  @Test
```
```suggestion
  @Test
  @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
```